.changeset/many-olives-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-techdocs': patch
+---
+
+Fix on shadow-dom selector
```

Comment:
Perhaps this could describe a bit more around what was fixed rather than how it was fixed?